### PR TITLE
PreviousClaims page

### LINKED ISSUE #3067: PreviousClaims page
There's now a previous claims page in the form: https://app.moqups.com/greg@adhocteam.us/l32hJ36b/view/page/abda3bffe

### PR DESCRIPTION
**Pull request by jbalboni**
Closes #3067.

Adds previous claims page. Also adds the question that triggers it to benefits selection.

### CODE REVIEW

**Comment by jbalboni** [comment]
@alexose Can you take a look at this?

File: src/js/edu-benefits/utils/helpers.js [hunk 1]
```diff
@@ -88,3 +88,8 @@ export function displayDateIfValid(dateObject) {
   }
   return null;
 }
+
+export function showSomeonElseServiceQuestion(claimType) {
```
Comment (alexose) [inline src/js/edu-benefits/utils/helpers.js]: Should this be showSomeon**e**ElseServiceQuestion ?

**Review by alexose — APPROVED** [review]
Looks good!

File: src/js/edu-benefits/utils/helpers.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/utils/helpers.js]: Yep. Fixed now.

**Pull request merged**